scripts/deploy_docs.sh

```diff
@@ -0,0 +1,27 @@
+DEPLOY_NIGHTLY_GITHUB_USER=leanprover-community-bot
+
+set -e
+set -x
+
+# By default, github actions overrides the credentials used to access any
+# github url so that it uses the github-actions[bot] user.  We want to access
+# github using a different username.
+git config --unset http.https://github.com/.extraheader
+
+git_hash="$(git log -1 --pretty=format:%h)"
```

Comment:
Maybe it's a good idea to clone all these repository outside of the `mathlib` directory.  Though it doesn't break anything right now. 
```
cd ..
```